games/snake.py

```diff
@@ -1,22 +1,26 @@
 import random
 import curses
 
+from collections import namedtuple
+
+Point = namedtuple('Point', ['x_pos', 'y_pos'])
```

Comment:
Let's use this alternative for this point: https://github.com/jnduli/python_curses_games/blob/3f04025ae26c8d6801b1c4e67f50c0d1467aa150/games/tetris/shapes.py#L3

I think we can place it on a parent class, perhaps Shape.py and import the same into the games

Reply:
For now, I've moved this to a shapes.py. My intention is that this file will evolve over time as we refactor other things.